packages/plugin-core/src/test/suite-integ/ConvertLink.test.ts

```diff
@@ -0,0 +1,374 @@
+import { NoteProps } from "@dendronhq/common-all";
+import {
+  CreateNoteOptsV4,
+  NoteTestUtilsV4,
+} from "@dendronhq/common-test-utils";
+import { LinkUtils, ParseLinkV2Resp } from "@dendronhq/engine-server";
+import _ from "lodash";
+import { beforeEach, afterEach } from "mocha";
+import sinon from "sinon";
+import vscode from "vscode";
+import { ConvertLinkCommand } from "../../commands/ConvertLink";
+import {
+  getReferenceAtPosition,
+  getReferenceAtPositionResp,
+} from "../../utils/md";
+import { VSCodeUtils } from "../../vsCodeUtils";
+import { WSUtils } from "../../WSUtils";
+import { expect } from "../testUtilsv2";
+import { describeMultiWS, setupBeforeAfter } from "../testUtilsV3";
+
+suite("ConvertLink", function () {
+  const ctx = setupBeforeAfter(this, {
+    beforeHook: () => {},
+  });
+
+  let activeNote: NoteProps;
+  let activeNoteCreateOpts: CreateNoteOptsV4;
+  let anotherNote: NoteProps;
+
+  const noAliasBrokenLinkPosition = new vscode.Position(7, 0);
+  const aliasBrokenLinkPosition = new vscode.Position(8, 0);
+
+  describeMultiWS(
+    "GIVEN note with broken links",
+    {
+      ctx,
+      preSetupHook: async (opts) => {
+        const { vaults, wsRoot } = opts;
+        activeNoteCreateOpts = {
+          fname: "active",
+          vault: vaults[0],
+          wsRoot,
+          body: [
+            "[[foo.bar.broken.link]]", // link 7
```

Comment:
would be good to also add these in `dendron.ref.links` under a `## Broken Links` section

Reply:
It's already there but I'll add a few more add some context to it